src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java

```diff
@@ -2012,163 +1972,6 @@ public void testIsFeedbackSessionFullyCompletedByStudent() throws Exception {
-                assertEquals(String.format(EmailType.FEEDBACK_SESSION_REMINDER.getSubject(), courseName,
-                                           fs.getFeedbackSessionName()),
-                             emailsToInstructor.get(0).getSubject());
-                assertEquals(String.format(EmailType.FEEDBACK_SESSION_REMINDER.getSubject(), courseName,
-                                           fs.getFeedbackSessionName()),
-                             emailsToInstructor.get(1).getSubject());
-            }
-
-        }
-        
-        ______TS("failure: non-existent Feedback session");
-        
-        String nonExistentFsName = "non-ExIsTENT FsnaMe123";
-        
-        try {
-            fsLogic.sendReminderForFeedbackSession(fs.getCourseId(), nonExistentFsName);
-            signalFailureToDetectException();
-        } catch (EntityDoesNotExistException edne) {
-            assertEquals("Trying to remind a non-existent feedback session: "
-                         + fs.getCourseId() + "/" + nonExistentFsName,
-                         edne.getMessage());
-        }
-        
-    }
-    
-    public void testSendReminderForFeedbackSessionParticularUsers() throws Exception {
-        // private method. no need to check for authentication.
-        Logic logic = new Logic();
-        
-        ______TS("typical success case");
-        
-        StudentAttributes studentToRemind = dataBundle.students.get("student5InCourse1");
-        InstructorAttributes instrToRemind = dataBundle.instructors.get("helperOfCourse1");
-        
-        FeedbackSessionAttributes fs = dataBundle.feedbackSessions.get("session1InCourse1");
-        String courseName = CoursesLogic.inst().getCourse(fs.getCourseId()).getName();
-        String[] usersToRemind = new String[] {studentToRemind.email, instrToRemind.email};
-
-        List<EmailWrapper> emailsSent =
-                fsLogic.sendReminderForFeedbackSessionParticularUsers(
-                        fs.getCourseId(), fs.getFeedbackSessionName(), usersToRemind);
-        assertEquals(7, emailsSent.size());
-
-        EmailWrapper emailToStudent = getEmailToStudent(studentToRemind, emailsSent);
-        String errorMessage = "No email sent to selected student " + studentToRemind.email;
-        assertNotNull(errorMessage, emailToStudent);
-        assertEquals(String.format(EmailType.FEEDBACK_SESSION_REMINDER.getSubject(), courseName,
-                                   fs.getFeedbackSessionName()),
-                     emailToStudent.getSubject());
-
-        List<InstructorAttributes> instructorList = logic.getInstructorsForCourse(fs.getCourseId());
-        String notificationHeader = "The email below has been sent to students of course: " + fs.getCourseId();
-        for (InstructorAttributes i : instructorList) {
-            List<EmailWrapper> emailsToInstructor = getEmailsToInstructor(i, emailsSent);
-            
-            if (i.email.equals(instrToRemind.email)) {
```

Comment:
Ok, got it. Remember to make issues for loss of details for this test then, or will this be done as part of #6223?